client/src/pages/Profile/index.js

```diff
@@ -1,183 +1,227 @@
-                            <img className = {classes.bannerphoto} src="https://dummyimage.com/500/09f/fff.png" alt="Mock Name"></img>
-                        </div>
-                        <h1 className = {classes.bannername}>Mock Name</h1>
-                    </div>
-            </div>
-        )
-    }
+class ProfileHeader extends Component {
+  render() {
+    return (
+      <div>
+        <div className={classes.prfileoverlay}></div>
+        <div className={classes.intro}>
+          <div className={classes.profileimage}>
+            <img
+              className={classes.bannerphoto}
+              src="https://dummyimage.com/500/09f/fff.png"
+              alt="Mock Name"
+            ></img>
+          </div>
+          <h1 className={classes.bannername}>Mock Name</h1>
+        </div>
+      </div>
+    );
+  }
 }
 
-class NavItem extends Component{
-    render(){
-        return(
-            <li><span className = {classes.item} id = {this.props.activeProfile && "active"} onClick={this.props.action}>{this.props.children}</span></li> 
-        )
-    }
+class NavItem extends Component {
+  render() {
+    return (
+      <li>
+        <span
+          className={classes.item}
+          id={this.props.activeProfile && "active"}
+          onClick={this.props.action}
+        >
+          {this.props.children}
+        </span>
+      </li>
+    );
+  }
 }
 
 class Profile extends Component {
-    constructor(props) {
-        super(props)
-        this.clickProfile = this.clickProfile.bind(this);
-        this.clickMeeting = this.clickMeeting.bind(this);
-        this.state = {activeProfile:true,isProfile: true, isMeeting:false}
-      }  
-    clickProfile(props){
-        this.setState({...this.state,activeProfile:true,isProfile: true, isMeeting:false})  
-    }
-    clickMeeting(props){
-        this.setState({...this.state,activeProfile:false,isProfile: false, isMeeting:true})  
-    }
-    render() {
-      return (
-        <div>  
-            <ProfileHeader/> 
-            <nav className = {classes.profilenav}>        
-                <ul className = {classes.nav}>
-                    <NavItem action={this.clickProfile} activeProfile = {this.state.activeProfile}>Profile</NavItem>
-                    <NavItem action={this.clickMeeting} activeProfile = {!this.state.activeProfile}>Meeting</NavItem>
-                </ul>
-            </nav>
-            {this.state.isMeeting && <MeetingContent/>}
-            {this.state.isProfile && <ProfileContent/>}
-        </div>
-      )
-    }
+  constructor(props) {
+    super(props);
+    this.clickProfile = this.clickProfile.bind(this);
+    this.clickMeeting = this.clickMeeting.bind(this);
+    this.state = { activeProfile: true, isProfile: true, isMeeting: false };
+  }
+  clickProfile(props) {
+    this.setState({
+      ...this.state,
+      activeProfile: true,
+      isProfile: true,
+      isMeeting: false,
+    });
```

Comment:
```suggestion
    this.setState({
      activeProfile: true,
      isProfile: true,
      isMeeting: false,
    });
```

Reply:
This isn't my code but i'll commit it anyway :P